test/services/bulkstatus.service.test.js

```diff
@@ -81,6 +80,19 @@ describe('Dynamic X-Progress logic', () => {
         expect(response.headers['x-progress']).toEqual('60.00% Done');
       });
   });
+  test('check operationOutcome includes the number of resources when available', async () => {
```

Comment:
Last comment. This test is currently in a `describe` block for `Dynamic X-Progress logic`, can we move it to the above `describe` block labeled `checkBulkStatus logic`?